tests/formats/fork_choice/README.md

```diff
@@ -0,0 +1,101 @@
+# Fork choice tests
+
+The aim of the tests for the fork choice rules.
+
+## Test case format
+
+### `meta.yaml`
+
+```yaml
+description: string    -- Optional. Description of test case, purely for debugging purposes.
```

Comment:
Are we sing this anywhere? or do we intend to?

Reply:
Per Proto's suggestion (https://github.com/ethereum/eth2.0-specs/pull/2202#discussion_r593161385), added description for `test_genesis`.